package.json

```diff
@@ -70,13 +71,15 @@
   },
   "dependencies": {
     "@tanstack/react-query": "^4.28.0",
-    "seamapi": "^5.15.0",
+    "seamapi": "^5.17.0",
     "uuid": "^9.0.0"
   },
   "devDependencies": {
     "@babel/preset-env": "^7.21.4",
     "@babel/preset-react": "^7.18.6",
     "@babel/preset-typescript": "^7.21.4",
+    "@emotion/styled": "^11.10.6",
```

Comment:
If we are using `emotion` and `material` do we need those as dependencies? And if we are brining those in, why have two style system (emotion and scss)?

Reply:
Only planning to use `material` (`emotion` is a MUI dep) in the storybook demos. Not the final bundle.

Reply:
Ooo, interesting, so like UI to control the components?

Reply:
Yea, just as examples.

Reply:
I may move the style toolchain to it's own PR. We should probably integrate this https://storybook.js.org/recipes/@emotion/styled